next/components/header.tsx

```diff
@@ -52,8 +52,15 @@ function SearchLink() {
   const { locale } = useRouter();
   const { t } = useTranslation();
   return (
-    <Link href="/search" locale={locale} className="hover:underline">
-      <span className="hidden sm:mr-2 sm:inline">{t("search")}</span>
+    <Link
+      href="/search"
+      locale={locale}
+      className="hover:underline"
+      aria-label={t("search-link")}
+    >
+      <span className="sr-only sm:not-sr-only sm:mr-2 sm:inline">
+        {t("search")}
+      </span>
       <SearchIcon className="inline-block h-6 w-6" />
     </Link>
```

Comment:
Here, I feel like aria-label should be unnecessary because the child span (which is always visible to screen readers) already tells us what it does?

If the automated a11y software says it IS needed, can we again use aria-describedby to point to the span, instead of adding another label with its translation etc?

Reply:
This has been removed now.